packages/web/src/components/audio-transaction-icon/AudioTransactionIcon.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { ReactNode } from 'react'
+
+import {
+  TransactionDetails,
+  TransactionMethod,
+  TransactionType
+} from '@audius/common'
+import cn from 'classnames'
+
+import AppIcon from 'assets/img/appIcon.png'
+import { ReactComponent as IconCoinbaseMini } from 'assets/img/iconCoinbaseMini.svg'
+import { ReactComponent as IconReceiveMini } from 'assets/img/iconReceive.svg'
+import { ReactComponent as IconSendMini } from 'assets/img/iconSend.svg'
+import { ReactComponent as IconStripeMini } from 'assets/img/iconStripeMini.svg'
+import { ReactComponent as IconTip } from 'assets/img/iconTip.svg'
+import { ReactComponent as IconTrophy } from 'assets/img/iconTrophy.svg'
+
+import styles from './AudioTransactionIcon.module.css'
+
+type IconProps = {
+  transactionDetails: TransactionDetails
+}
+
+const typeIconSvgMap: Record<
+  TransactionType,
+  React.FunctionComponent<React.SVGProps<SVGSVGElement>> | null
+> = {
+  [TransactionType.CHALLENGE_REWARD]: IconTrophy,
+  [TransactionType.PURCHASE]: null, // Not needed, AppLogo is used for purchases
+  [TransactionType.TIP]: IconTip,
+  [TransactionType.TRANSFER]: null, // TODO: Add icon for this
+  [TransactionType.TRENDING_REWARD]: IconTrophy
+} as const
+
+const AppLogo = () => (
+  <img src={AppIcon} alt={'Audius Logo'} width={40} height={40} />
+)
+
+const TypeIcon = ({ transactionDetails }: IconProps) => {
```

Comment:
Maybe make the props explicitly the `method` and `type` instead, since that's all it uses?